src/core/components/init-assets.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs')
+const glob = require('glob')
+const importer = require('ipfs-unixfs-engine').importer
+const pull = require('pull-stream')
+const file = require('pull-file')
+const mh = require('multihashes')
+
+// Add the default assets to the repo.
+module.exports = function addDefaultAssets (self, log, callback) {
+  const initDocsPath = path.join(__dirname, '../../init-files/init-docs')
+  const index = __dirname.lastIndexOf('/')
+
+  pull(
+    pull.values([initDocsPath]),
+    pull.asyncMap((val, cb) => {
+      glob(path.join(val, '/**/*'), cb)
+    }),
+    pull.flatten(),
+    pull.map((element) => {
+      const addPath = element.substring(index + 1, element.length)
+      if (fs.statSync(element).isDirectory()) return
+
+      return {
+        path: addPath,
+        content: file(element)
+      }
+    }),
+    pull.filter(Boolean),
```

Comment:
It filters out all `undefined` elements, which are directories, if you check a couple of lines above